nerdlandbot/eventhandlers/oncommanderror.py

```diff
@@ -2,9 +2,12 @@
 
 from discord.ext import commands
 
+
 from nerdlandbot.translations.Translations import get_text as translate
-from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture_ctx
```

Comment:
would keep the original `culture` import since it would remain consistent with all other files in the repo.